packages/test-generator/lib/generators/TestGenerator.ts

```diff
@@ -49,8 +50,8 @@ export type TestCase = {
 export type TestGeneratorParams = {
   writeToLogger: boolean;
   writeToDisk: boolean;
-  renderConfigOverride?: ReactRenderConfig;
-  outputConfigOverride?: ReactOutputConfig;
+  renderConfigOverride?: Partial<ReactRenderConfig>;
```

Comment:
Is this necessary to convert it to Partial? I'd prefer it to check for the required types when writing new tests. We already have DEFAULT_RENDER_CONFIG defined above. Can we not just add all the base config there for ease of use?

Reply:
`DEFAULT_RENDER_CONFIG` did have the required property added to it, the issue is that the overrides should not be required to have all properties set which is why I had previously added the `Partial` to them. If you look at lines 66-67 where these override parameters are referenced you'll see that the use of `Partial` matches the intent as we spread the overrides over the defaults, but not having it was never a problem until recently because all the properties were optional.